sunpy/cm/cm.py

```diff
@@ -168,7 +169,7 @@ def show_colormaps(filter=None):
     """
 
     if filter:
-        maps =  sorted({k:v for (k,v) in cmlist.iteritems() if k.lower().count(filter.lower())})
+        maps =  sorted({k:v for (k,v) in cmlist.items() if k.lower().count(filter.lower())})
```

Comment:
@dpshelio @nabobalis I removed the six iteritems hack here. The downside is that it uses the list rather than the generator implementation in Python 2, however the advantage is that it is natural syntax in both langs, and doesn't need six.